release_creation/bundle/requirements/v1.5.pre.requirements.txt

```diff
@@ -1,13 +1,15 @@
---pre
-dbt-core~=1.5.0b1 --no-binary dbt-postgres
-dbt-snowflake~=1.5.0b1
-dbt-bigquery~=1.5.0b1
-dbt-redshift~=1.5.0b1
-dbt-postgres~=1.5.0b1
-dbt-spark[PyHive,ODBC]~=1.5.0b1
+git+https://github.com/dbt-labs/dbt-core.git@1.5.latest#egg=dbt-core&subdirectory=core --no-binary dbt-postgres
+git+https://github.com/dbt-labs/dbt-core.git@1.5.latest#egg=dbt-postgres&subdirectory=plugins/postgres
+git+https://github.com/dbt-labs/dbt-bigquery.git@1.5.latest#egg=dbt-bigquery
+git+https://github.com/dbt-labs/dbt-snowflake.git@1.5.latest#egg=dbt-snowflake
+git+https://github.com/dbt-labs/dbt-redshift.git@1.5.latest#egg=dbt-redshift
+git+https://github.com/dbt-labs/dbt-spark.git@1.5.latest#egg=dbt-spark[PyHive,ODBC]
+dbt-databricks~=1.5.0
```

Comment:
I understand why we're not pulling these directly from GitHub. But should we add --pre or the prerelease suffix here to allow, for instance, dbt-databricks==1.5.1b1?